cloud/jenkins/pxc_operator_gke_version.groovy

```diff
@@ -239,6 +251,14 @@ pipeline {
         PXC_TAG = sh(script: "if [ -n \"\${IMAGE_PXC}\" ] ; then echo ${IMAGE_PXC} | awk -F':' '{print \$2}'; else echo 'main'; fi", , returnStdout: true).trim()
     }
     parameters {
+        choice(
+            choices: ['run-release.csv', 'run-distro.csv', 'run-minikube.csv', 'run-pr.csv'],
```

Comment:
Why do we need to have 'run-minikube.csv' for gke job?

Reply:
The intention was to put all the files which exist, nothing else. I will remove minikube from non-minikube, but should we remove the run-pr.csv as well since these jobs will not be used for PR runs?